clients/java/signalr/src/main/java/com/microsoft/signalr/HubMessageType.java

```diff
@@ -10,7 +10,8 @@
     STREAM_INVOCATION(4),
     CANCEL_INVOCATION(5),
     PING(6),
-    CLOSE(7);
+    CLOSE(7),
+    INVOCATIONBINDINGFAILURE(-10);
```

Comment:
I don't think we should add a dedicated message for this.

Reply:
We need a message type here, like in .NET, because we need to know the invocation ID and only the parser knows that. We use a pseudo-message to communicate the invocation ID and the binding error back to the caller of the parser.